desktop/src/main/java/bisq/desktop/main/portfolio/bsqswaps/UnconfirmedBsqSwapsView.java

```diff
@@ -242,25 +231,23 @@ protected void activate() {
             };
             CSVEntryConverter<UnconfirmedBsqSwapsListItem> contentConverter = item -> {
                 String[] columns = new String[ColumnNames.values().length];
-                columns[ColumnNames.TRADE_ID.ordinal()] = model.getTradeId(item);
-                columns[ColumnNames.DATE.ordinal()] = model.getDate(item);
-                columns[ColumnNames.MARKET.ordinal()] = model.getMarketLabel(item);
-                columns[ColumnNames.PRICE.ordinal()] = model.getPrice(item);
-                columns[ColumnNames.AMOUNT.ordinal()] = model.getAmount(item);
-                columns[ColumnNames.VOLUME.ordinal()] = model.getVolume(item);
-                columns[ColumnNames.TX_FEE.ordinal()] = model.getTxFee(item);
-                columns[ColumnNames.TRADE_FEE.ordinal()] = model.getTradeFee(item);
-                columns[ColumnNames.OFFER_TYPE.ordinal()] = model.getDirectionLabel(item);
-                columns[ColumnNames.CONF.ordinal()] = String.valueOf(model.getConfidence(item));
+                columns[ColumnNames.TRADE_ID.ordinal()] = item.getTradeId();
+                columns[ColumnNames.DATE.ordinal()] = item.getDateAsString();
+                columns[ColumnNames.MARKET.ordinal()] = item.getMarketLabel();
+                columns[ColumnNames.PRICE.ordinal()] = item.getPriceAsString();
+                columns[ColumnNames.AMOUNT.ordinal()] = item.getAmountAsString();
+                columns[ColumnNames.VOLUME.ordinal()] = item.getVolumeAsString();
+                columns[ColumnNames.TX_FEE.ordinal()] = item.getTxFeeAsString();
+                columns[ColumnNames.TRADE_FEE.ordinal()] = item.getTradeFeeAsString();
+                columns[ColumnNames.OFFER_TYPE.ordinal()] = item.getDirectionLabel();
+                columns[ColumnNames.CONF.ordinal()] = String.valueOf(item.getConfidence());
                 return columns;
             };
 
             GUIUtil.exportCSV("bsqSwapHistory.csv", headerConverter, contentConverter,
-                    new UnconfirmedBsqSwapsListItem(), sortedList, (Stage) root.getScene().getWindow());
+                    null, sortedList, (Stage) root.getScene().getWindow());
```

Comment:
As you changed it here and in other places. Do you now why an empty object was used before? 
:point @chimp1984

Reply:
It is passed into [`headerConverter`](https://github.com/xyzmaker123/bisq/blob/9e753458e4e248ac9b0852e05112b79c9dac42c1/desktop/src/main/java/bisq/desktop/util/GUIUtil.java#L330), but `headerConverter` doesn't use it, so passing null doesn't breaks anything. I think we could refactor `exportCSV` a bit deeper - I was planning to do it in dedicated pr.

Reply:
> Do you now why an empty object was used before? That was required for the exportCSV code... dont remember details but i think with null there was issues

Reply:
I tested export csv for bsq swaps with null passed as `emptyItem` and everything went smoothly.

Reply:
It also worked for me. I was just curious why it was needed in the past. Let's keep an eye on this.